kyo-bench/src/main/scala/kyo/bench/Bench.scala

```diff
@@ -25,24 +25,13 @@ import zio.UIO
 @BenchmarkMode(Array(Mode.Throughput))
 abstract class Bench[T](val expectedResult: T):
 
-    @Param(Array("false"))
-    var replaceZioExecutor = false
```

Comment:
I've moved this config to a system property to make it easier to compare benchmark results. The JMH visualizer considers a benchmark not the same if there's any difference in configuration between runs.